disnake/webhook/async_.py

```diff
@@ -975,13 +975,15 @@ def guild(self) -> Optional[Guild]:
         return self._state and self._state._get_guild(self.guild_id)
 
     @property
-    def channel(self) -> Optional[Union[TextChannel, VoiceChannel, ForumChannel, StageChannel]]:
-        """Optional[Union[:class:`TextChannel`, :class:`VoiceChannel`, :class:`ForumChannel`, :class:`StageChannel`]]: The channel this webhook belongs to.
+    def channel(
+        self,
+    ) -> Optional[Union[TextChannel, VoiceChannel, StageChannel, ForumChannel, MediaChannel]]:
+        """Optional[Union[:class:`TextChannel`, :class:`VoiceChannel`, :class:`StageChannel`, :class:`ForumChannel`, :class:`MediaChannel`]]: The channel this webhook belongs to.
 
         If this is a partial webhook, then this will always return ``None``.
 
-        Webhooks in :class:`ForumChannel`\\s can not send messages directly,
-        they can only create new threads (see ``thread_name`` for :attr:`Webhook.send`)
+        Webhooks in channels of types :class:`ForumChannel' or :class:`MediaChannel` can not send messages directly.
```

Comment:
the UI doesn't seem to allow the creation of webhooks 🤔

Reply:
asked in ddevs https://canary.discord.com/channels/613425648685547541/697489244649816084/1120071900954108084

Reply:
and on the github issue https://github.com/discord/discord-api-docs/pull/6232#issuecomment-1596245447

Reply:
this was fixed yesterday, webhooks are now shown in the UI as well